Tests/features/CuckooGeneratorTests.feature

```diff
@@ -0,0 +1,5 @@
+Feature: CuckooGeneratorTests
+	Scenario: Generating test mocks
+		Given I run `xcodebuild -project '../../../CuckooGenerator.xcodeproj' -scheme 'CuckooGenerator' -configuration 'Release' CONFIGURATION_BUILD_DIR=Tests/tmp/aruba clean build`
```

Comment:
This is really too verbose and should not be part of the test. Cuckoo should be already built before testing starts. We definitely don't want to build it in each scenario.